packages/devtools_app/lib/src/memory/flutter/memory_heatmap.dart

```diff
@@ -106,12 +242,27 @@ class FlameChartRenderObject extends RenderBox {
     return null;
   }
 
+  void externalSelectNode(Node value) {
+    selectedNode = value;
+
+    // TODO(terry): Need to force a relayout to repaint (paintAncestors) with
+    //              new selection. Calling debugResetSize or size changing
+    //              works.  However, shouldn't hitTestSelf do the same e.g.,
+    //
+    //                  hitTestSelf(Offset(
+    //                    _selectedNode.rect.left + paintOffset.dx,
+    //                    _selectedNode.rect.top - paintOffset.dy,
+    //                  ));
+    size = size;
```

Comment:
does calling https://api.flutter.dev/flutter/rendering/RenderObject/markNeedsLayout.html
work? setting the size to itself is a little surprising and might stop having the desired effect if some optimization was added that detected that the size was unchanged.

Reply:
I've tried all of these:

      markNeedsLayout();
      markParentNeedsLayout();
      markNeedsLayoutForSizedByParentChange();
      markNeedsSemanticsUpdate();
      markNeedsCompositingBitsUpdate();

Nothing but size=size works but not great.  There is a TODO around this need to ask the Flutter folks for suggestions.  Maybe custom painter is a solution?

Reply:
a TODO is fine. Maybe file a bug because this is quite odd. perhaps there is something strange about when we are calling externalSelectNode.

Reply:
Done.